docs/html/dropdown/dropdown--with-prepend.html

```diff
@@ -0,0 +1,27 @@
+<div className="ray-dropdown ray-dropdown--with-prepend">
+    <div className="ray-dropdown__prepend">
+      <div>🇺🇸</div>
+    </div>
+    <div className="ray-dropdown__wrapper">
+      <select
+        className="ray-dropdown__input"
+        id="test-example"
+        value=""
+        required
+        aria-required="true"
+      >
+        <option value="" data-ray-placeholder>
+          Choose pokemon?
+        </option>
+        <option value="Slowpoke">Slowpoke</option>
+        <option disabled data-ray-separator />
+        <option value="Pikachu">Pikachu</option>
+        <option value="Squirtle">Squirtle</option>
+        <option value="Charmander">Charmander</option>
+        <option value="Sonichu">Sonichu</option>
+      </select>
+      <label className="ray-dropdown__label" htmlFor="test-example">
+        Pokemon
+      </label>
+    </div>
+  </div>
```

Comment:
Make sure all HTML is valid (not JSX) and please fix the indentation

Reply:
done